bazel/thirdparty/krb5.BUILD

```diff
@@ -48,12 +54,10 @@ configure_make(
         "--disable-static",
         "--enable-asan=$(SANITIZERS)",
     ],
-    # Need to pass this additionally here because of a bug in the kerberos build where it doesn't properly pass the linker flag down
-    copts = [
-        "-fuse-ld=lld",
-    ],
     env = {
+        # Need to pass this additionally here because of a bug in the kerberos build where it doesn't properly pass the linker flag down
         "KRB5_BUILD_JOBS": "$(BUILD_JOBS)",
+        "LD": gold,
```

Comment:
I don't know how to make this work with bazel, I wanted this line to be:
```suggestion
        "LD": "$(LINKER)",
```
But it won't expand there, apparently.

Reply:
you need to add the liner flag to toolchains for that to work

Reply:
> you need to add the linker flag to toolchains for that to work

I don't know why this version was pushed, but I had actualyl set
```suggestion
        "LD": "gold",

Reply:
> I don't know what you mean with the toolchain, but everything I tried didn't work. The configure script failed to pick up the correct linker.

Does the commit I just pushed work for you?

Reply:
> > I don't know what you mean with the toolchain, but everything I tried didn't work. The configure script failed to pick up the correct linker.
> 
> Does the commit I just pushed work for you?

You are my hero.